proof/courier.go

```diff
@@ -891,8 +886,7 @@ func (h *HashMailCourier) publishSubscriberEvent(event fn.Event) {
 // Close closes the underlying connection to the hashmail server.
 func (h *HashMailCourier) Close() error {
 	if err := h.mailbox.Close(); err != nil {
-		log.Warnf("unable to close mailbox session, "+
-			"recipient=%v: %v", err, spew.Sdump(h.recipient))
+		log.Warnf("unable to close mailbox session: %v", err)
```

Comment:
micro-nit (since we're touching the line anyway): Log messages should start capitalized.